gluten-iceberg/src/test/scala/io/glutenproject/execution/VeloxTPCHIcebergSuite.scala

```diff
@@ -84,3 +93,21 @@ class VeloxTPCHIcebergSuite extends VeloxTPCHSuite {
     }
   }
 }
+
+class VeloxPartitionedTableTPCHIcebergSuite extends VeloxTPCHIcebergSuite {
```

Comment:
@yma11 Added a test case that will execute all TPCH tests under a partitioned iceberg table.